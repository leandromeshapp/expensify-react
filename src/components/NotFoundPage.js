import React from "react"
import { Link } from "react-router-dom"
const NotFoundPage = () => (
    <div>
        404 NOT FOUND
        <Link to ="/">
            HOME
        </Link>
    </div>
)

export default NotFoundPage