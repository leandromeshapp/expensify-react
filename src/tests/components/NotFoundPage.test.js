import React from "react"
import { shallow } from "enzyme"
import NotFoundPage from "../../components/NotFoundPage"


// Render Not Found Page Component
test("Should render NotFoundPage correctly", () => {
    const wrapper = shallow(<NotFoundPage />)

    expect(wrapper).toMatchSnapshot()
})
