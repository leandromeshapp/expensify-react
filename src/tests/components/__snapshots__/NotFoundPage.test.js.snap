// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotFoundPage correctly 1`] = `
<div>
  404 NOT FOUND
  <Link
    replace={false}
    to="/"
  >
    HOME
  </Link>
</div>
`;
